.travis.yml

```diff
@@ -14,249 +14,262 @@
+    fi
 
-       brew tap FreeCAD/freecad
+    export CMAKE_ARGS="${CMAKE_OPTS} -DFREECAD_USE_EXTERNAL_KDL=ON -DEIGEN3_INCLUDE_DIR=$(brew --prefix eigen@3.2)/include/eigen3"
+- &gcc_compiler
+  compiler: gcc
+- &clang_compiler
+  compiler: clang
 
-       if [ "${QT}" == "Qt4" ]; then
-          # Qt4: Replace Qt5 from ports-cache with Qt4, if necessary
-          brew ls --versions qt@5.6 > /dev/null && \
-             brew uninstall --ignore-dependencies --force shiboken pyside pyside-tools qt@5.6
+addons:
+  apt:
+    packages:
+    - doxygen  # docs generator
+    - libboost1.55-dev
+    - libboost-filesystem1.55-dev
+    - libboost-program-options1.55-dev
+    - libboost-python1.55-dev
+    - libboost-regex1.55-dev
+    - libboost-signals1.55-dev
+    - libboost-system1.55-dev
+    - libboost-thread1.55-dev
+    - libcoin80
+    - libcoin80-dev
+    - liboce-foundation-dev
+    - liboce-modeling-dev
+    - liboce-ocaf-dev
+    - liboce-ocaf-lite-dev
+    - liboce-visualization-dev
+    - libpyside-dev
+    - libqtcore4
+    - libshiboken-dev
+    - libxerces-c-dev
+    - libxmu-dev
+    - libxmu-headers
+    - libxmu6
+    - libxmuu-dev
+    - libxmuu1
+    - netgen
+    - netgen-headers
+    - oce-draw
+    - pyside-tools
+    - python-dev
+    - python-pyside
+    - python-matplotlib
+    - python3-dev
+    - python3-pyside
+    - python3-matplotlib
+    - qt4-dev-tools
+    - qt4-qmake
+    - libqt4-opengl-dev
+    - shiboken
+    - swig
+    - libvtk6-dev
+    - libmed-dev
+    - libmedc-dev
+    - asciidoc
 
-          brew tap cartr/qt4
-
-          brew install --verbose --only-dependencies freecad --with-qt4 --with-packaging-utils
-          brew install --verbose --only-dependencies freecad --with-qt4 --with-packaging-utils #Ensure all dependencies are satisfied
-          # qt4 bottle contains postgresql plugin even when --with-postgresql option is not specified
-          # and so postgresql dep is not installed. Delete it because the plugin is not included with qt5 bottle.
-          sudo rm -f /usr/local/lib/qt4/plugins/sqldrivers/libqsqlpsql.dylib
-       elif [ "${QT}" == "Qt5" ]; then
-          # Qt5: Replace Qt4 in ports-cache with Qt5, if necessary
-          brew ls --versions cartr/qt4/qt > /dev/null && \
-             brew uninstall --ignore-dependencies --force cartr/qt4/shiboken@1.2 cartr/qt4/pyside@1.2 cartr/qt4/pyside-tools@1.2 cartr/qt4/qt
-
-          brew install --verbose --only-dependencies freecad --with-packaging-utils
-          brew install --verbose --only-dependencies freecad --with-packaging-utils #Ensure all dependencies are satisfied
-
-          # Qt5:  Set Qt5 build flag and CMAKE_PREFIX
-          QT5_CMAKE_PREFIX=$(ls -d $(brew list -1 | grep qt | tail -1 | xargs brew --cellar)/*/lib/cmake)
-          CMAKE_OPTS="${CMAKE_OPTS} -DBUILD_QT5=ON -DCMAKE_PREFIX_PATH=${QT5_CMAKE_PREFIX}"
-       fi
+cache:
+  ccache: true
+  directories:
+  - $HOME/Library/Caches/Homebrew
```

Comment:
This is probably redundant with FreeCAD/Ports-cache will less of a positive impact on the builds.  The FreeCAD/Ports-cache caches the homebrew ports and restores them in one fell swoop.  However, if you know of a better way to cache the brew ports, then let's do it!

Reply:
how does your helper differs from Travis' native cache?

Reply:
Take at look at the ReadMe here - https://github.com/FreeCAD/FreeCAD-ports-cache 

Bottom-line, it caches /usr/local/Homebrew (that actual brewed ports) vs. the Homebrew cache that contains just the source tarballs and/or bottles.